### ISSUE TITLE
Bug: `this hasn't been initialised - super() hasn't been called`

### ISSUE THREAD

**Issue by garrettjstevens** [GMOD/jbrowse-components#1298]
Currently if I go the the deployed master branch at https://s3.amazonaws.com/jbrowse.org/code/jb2/alpha/master/index.html and do `File -> Add -> Linear genome view`, it crashes with the following error:

```
rpc.worker.ts:60 rpc-error 1 CoreGetRegions ReferenceError: this hasn't been initialised - super() hasn't been called
    at r (assertThisInitialized.js:3)
    at new n (IndexedFastaAdapter.ts:25)
    at n.<anonymous> (createSuper.js:11)
    at new n (BgzipFastaAdapter.ts:11)
    at dr (dataAdapterCache.ts:62)
    at n.<anonymous> (coreRpcMethods.ts:28)
    at u (runtime.js:63)
    at Generator._invoke (runtime.js:293)
    at Generator.next (runtime.js:118)
    at r (asyncToGenerator.js:3)
    at s (asyncToGenerator.js:25)
mg @ rpc.worker.ts:60
(anonymous) @ rpc.worker.ts:95
Promise.catch (async)
(anonymous) @ rpc.worker.ts:79
Promise.then (async)
b.handler @ web-rpc.js:321
```

If I build and serve master locally, though, this error does not happen. It only seems to happen on the S3 hosted build.

**Comment by garrettjstevens** [comment]
Tried to fix in #1293, but that didn't work.

**Comment by cmdcolin** [comment]
Here is a link straight to the failure http://s3.amazonaws.com/jbrowse.org/code/jb2/alpha/master/index.html?config=test_data%2Fconfig_demo.json&session=eJztVl1v2jAU_SvIT5tEk0AolLy1ndZWYx0C1q6bpsgkl8RtYke2KVDEf9-1w0K60mrTJk2T9pCH3K9z7o1PrteE0xxIQC5h0RiDUkzwRttrewde_8BrTVqdwOsHftc56rU-kybJqUwYJ4HXJLGkC5DXLNYpCfyjTpPcM1goEnxZExZjyatwcDcfCRhiXsxUkdHVZQmWJq0-GvWqMG8DxoHKM-AihyssgR4xmynQwyUJWl6r33N8_9Dvdb12p93pNsm0GIK0zqowxCNIkHsJL2G2RWphMaWp1JYycOTV7vTbh163jckS7kEqQOOMZgqahCoF-TR7xHPzFZlKGt3VWktvVm8OBzf9cNfFccYSngPXamKC0ZECS1IERrQmiQSfsWQuqUaWGM6TPJMhcmdaIwcMH7IM5kWZHGxxHj51L2adg9vl1Q6oHrcDaXlPQdb7U2wzF_FTEmFh48Ll6gHDtgdj_O5k5DcKGk2ZaLy6PHs_GL1uVCkNwbNVo8wjj-dnxlVOEAcYUQ2JkCtjsxWNkca0wCI1oic0P94a8TPTfCCiqpe5ZGbyWhcqcF3lOzSnD4LThXIikbu3UykWChwhEzexZ0m5Bt1Vd1PpuxJorELTrXl1ZjS0zR94Ttvxw5wWBcRO1ZaD2JaBgo8VLKJmyCdLhdKB73meq0HpMKaauuXkwxhy4dwqJLxpEsZjWBrm2V_sAh_2m51sNkYp2Iw0BxV7KI_TaGt6cs4qB-aN75O3QPVcwp7wPc6NRRtfDk8FSpMm8FgPV8VU9h6uj7s7OTyJ3Wmi84Ik9qQ9rwvFi2gb_OviqCX_AYXUeO-UoubT_1r6R7T0o5hqH3SfRPZ4bQ0FGUTarD1jZTzBYDMolYrF9wwSaDmH0lZKs7SYjZayGM5xAgZtu_x2pg-Yb5Z55bLSGFtIIScltZQhiIxShs2TEsQqaUCnkKk69iluRZBmzW8LGgILFieg7QjqlSZ1pEr3z0dU_4Hz50KuLRAxv4Iso4Vd92uc1IxlyGoCS21GV95eHt1azJhppNk9XP8U1xd8G_sj4viCR_eC442ER0b-683mG04iS1k
